.github/workflows/on-push.yml

```diff
@@ -35,62 +35,61 @@ jobs:
       - name: Install Dependencies
         if: steps.cache.outputs.cache-hit != 'true'
         run: yarn install --frozen-lockfile
-      - name: Check changes "docs"
-        id: docs-changed
-        run: echo "docs-changed=$(node tools/scripts/ci-is-affected.mjs --appName=docs --base=${{ env.BASE_REF}} --head=HEAD)" >> $GITHUB_OUTPUT
+      - name: Check changes "website"
+        id: website-changed
+        run: echo "website-changed=$(node tools/scripts/ci-is-affected.mjs --appName=website,docs,docs-ui --base=${{ env.BASE_REF}} --head=HEAD)" >> $GITHUB_OUTPUT
```

Comment:
you may discard this change and change to "ci-is-affected" if "implicitDependencies" param is set properly as per my other comment